stacked-predictions/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Stacked prediction",
+  "description": "Creates several kinds of predictors, runs them all on the same dataset, and appends the dataset with the most popular prediction",
+  "kind": "script",
```

Comment:
same nit as before: i at least am not sure of what "appending a dataset" means? not a native speaker, but i'd have said something like "and appends to the original dataset the most popular prediction" or "with the most popular prediction appended/added"